bin/scrape_software_versions.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import print_function
 from collections import OrderedDict
 import re
+import csv
```

Comment:
This is no longer needed, so I think you can remove this line again...

Reply:
Right! Thanks @ewels